website/content/en/v0.27.2/concepts/settings.md

```diff
@@ -41,7 +41,7 @@ data:
   # The maximum length of a batch window. The longer this is, the more pods we can consider for provisioning at one
   # time which usually results in fewer but larger nodes.
   batchMaxDuration: 10s
-  # The maximum amount of time with no new ending pods that if exceeded ends the current batching window. If pods arrive
+  # The maximum amount of time with no new pending pods that if exceeded ends the current batching window. If pods arrive
```

Comment:
Can you add this to `/preview/concepts` which will get picked up in the next release?

Reply:
oops.. done.